script.js

```diff
@@ -1,52 +1,129 @@
+    type: 'nature',
+  },
+  {
+    url: 'https://images.unsplash.com/photo-1509316975850-ff9c5deb0cd9?q=80&w=987&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D',
+    type: 'nature',
+  },
+  {
+    url: 'https://images.unsplash.com/photo-1477959858617-67f85cf4f1df?q=80&w=944&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D',
+    type: 'city',
+  },
+  {
+    url: 'https://images.unsplash.com/photo-1519501025264-65ba15a82390?q=80&w=1064&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D',
+    type: 'city',
+  },
+  {
+    url: 'https://images.unsplash.com/photo-1496568816309-51d7c20e3b21?q=80&w=1031&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D',
+    type: 'city',
+  },
+  {
+    url: 'https://images.unsplash.com/photo-1494522855154-9297ac14b55f?q=80&w=1170&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D',
+    type: 'city',
+  },
+  {
+    url: 'https://plus.unsplash.com/premium_photo-1669725687221-6fe12c2da6b1?q=80&w=987&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D',
+    type: 'animals',
+  },
+  {
+    url: 'https://plus.unsplash.com/premium_photo-1666278379770-440439b08656?q=80&w=988&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D',
+    type: 'animals',
+  },
+  {
+    url: 'https://images.unsplash.com/photo-1441057206919-63d19fac2369?q=80&w=987&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D',
+    type: 'animals',
+  },
+  {
+    url: 'https://images.unsplash.com/photo-1535268647677-300dbf3d78d1?q=80&w=987&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D',
+    type: 'animals',
+  },
 ];
 
-console.log('hello');
\ No newline at end of file
+let num = 5;
+
+function getImages() {
+  const gallery = document.getElementById('gallery');
+
+  for (let i = 0; i < photos.length; i++) {
+    const createImg = document.createElement('img');
+    createImg.src = photos[i].url;
+    createImg.classList.add(photos[i].type);
+    gallery.appendChild(createImg);
+
+    if (i > num) {
+      createImg.style.display = 'none';
+    }
+  }
+}
+
+function getAll() {
+  const allImages = document.querySelectorAll('img');
+
+  for (let i = 0; i < allImages.length; i++) {
+    if (i <= num) {
+      allImages[i].style.display = 'block';
+    } else {
+      allImages[i].style.display = 'none';
+    }
+  }
+}
+
+function getEach(buttonName) {
+  const allImages = document.querySelectorAll('img');
+
+  allImages.forEach((images) => {
+    if (images.className === buttonName) {
+      images.style.display = 'block';
+    } else {
+      images.style.display = 'none';
+    }
+  });
+}
+
+function showMore() {
+  num = num + 6;
```

Comment:
This is looking good, although one thing I noticed while testing your pages is that when you click the "Show more" button after one of the filters is selected, it add unrelated photos to your page. You should look into how you can fix that.